### ISSUE TITLE
Providing options to the user to upload the wallpaper, logo or the custom files

### ISSUE THREAD

**Issue by pranav1698** [fossasia/meilix-generator#439]
**I'm submitting a ...**  (check one with "x")
- [ ] bug report
- [x] feature request
- [ ] support request => Please do not submit support requests here, instead ask your query in out Gitter channel at https://gitter.im/fossasia/meilix

**Expected Behaviour**
Currently, the webapp only provides the user the option to upload a binary and then we convert it into a hosted url which is then used in the build server, many reviewers have asked to provide an option the user to get url where the wallpaper is hosted so that it can be directly downloaded into the ISO. 

**Would you like to work on the issue?**
Yes

**Comment by tabesin** [comment]
I do not fully understand the description.

You mean URL instead of local file?

**Comment by pranav1698** [comment]
Yes @tabesin, we can give an option to the users to add the URL of their hosted wallpaper, like imgur

**Comment by xeon-zolt** [comment]
so you are thinking of downloading and then sending the image or simply sending link if sending link then you need to update the meilix script too
